src/fullyautomatednutcracker/cogs/antiselfdeprecation.py

```diff
@@ -29,5 +32,16 @@ async def on_message(self, message):
         if message.content.lower() == 'yoshi man good':
             await message.add_reaction('\U0001F49A')
 
+    # bump timer, waits 2 hours and 30 minutes
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author.id == 302050872383242240 and 'Bump done' in message.embeds[0].description:
```

Comment:
check if the message has an embed first, just in case disboard is is asked for help or something else...

if author is disboard and if message has embeds and "bump done" in embed description...